oauth2client/pkce.py

```diff
@@ -0,0 +1,30 @@
+# Copyright 2016 Google Inc. All rights reserved.
```

Comment:
We could move these functions to `utils`, if you like that better. I dislike generic `utils` modules, though.